yt/frontends/enzo/answer_testing_support.py

```diff
@@ -122,8 +121,8 @@ def get_analytical_solution(self):
         pos, dens, vel, pres, inte = np.loadtxt(self.solution_file, unpack=True)
         exact = {}
         exact["pos"] = pos
```

Comment:
@jwise77 should this key be updated too or is there a reason not to ?

Reply:
It shouldn't be updated. This key is only used for interpolation purposes in the answer testing and not directly compared to the field data.

Reply:
fair enough !